mod-bulk-edit/src/main/resources/firebird/bulk-edit/features/bulk-edit-items-status.feature

```diff
@@ -19,10 +19,14 @@ Feature: bulk-edit items update status tests
     Given path 'data-export-spring/jobs'
     And headers applicationJsonContentType
     And request itemIdentifiersJob
+    * def Thread = Java.type('java.lang.Thread')
+    * Thread.sleep(300000)
```

Comment:
* call pause 10000 
This construction can be used as alternative to Thread.sleep in karate tests